### DISCUSSION TITLE
New person who wants to play, but doesn't know how to install easily.

### DISCUSSION THREAD

**Discussion by Cheesey123** [DarkflameUniverse/DarkflameServer#1190] [Q&A] [answered]
I would like to play, but I'm stupid, so I don't really know how to install it. Is there an easy tutorial I can follow so I can start playing?

**Comment by MustacheMan2** [comment, marked as answer]
This is about as easy as it gets, assuming you are on windows. https://gist.github.com/HailStorm32/169df65a47a104199b5cc57d10fa57de

**Reply by Cheesey123** [reply]
Thank you! I really am stupid at setting up game servers, so I might not be able to do this, but I appreciate it!
